lib/k8s-test-framework/src/framework.rs

```diff
@@ -68,6 +68,12 @@ impl Framework {
         exec_tail(&self.interface.kubectl_command, namespace, resource, file)
     }
 
+    /// Exect a `kubectl --version`command returning a json object
+    /// with all version information available for parsing
+    pub async fn version(&self) -> Result<version::K8sVersion> {
```

Comment:
```suggestion
    pub async fn kubernetes_version(&self) -> Result<version::K8sVersion> {
```

I think it is more meaningful to explicitly mention that this is about the Kubernetes version in the signature.